scripts/embed-dashboard-ui.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+BASE_DIR="$(dirname "$DIR")"
+CACHE_DIR=${BASE_DIR}/scripts/.dashboard_asset_cache
+
+echo '+ Create cache directory'
+mkdir -p ${CACHE_DIR}
+
+echo '+ Fetch Dashboard Go module'
+go mod download
```

Comment:
We are using retool now and should use the go mod as soon as possible. (No modification required)

Reply:
Yes! Looks like the official recommended way is to create a tools.go file to record the version of the tools, as is in TiDB Dashboard.

Reply:
Yes, I plan to remove retool, can do not do it in this PR.